TYPO3.TYPO3CR/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -65,6 +65,16 @@ additionalProperties:
             # we intentionally do not specify which properties are allowed here;
             # as other packages (such as TYPO3.Neos) might do that instead.
 
+          'options':
+            type: dictionary
+            # we intentionally do not specify which properties are allowed here;
+            # as other packages might do that instead.
```

Comment:
this is not used in the core AFAIK, thus doesn't really belong here.. but you could argue it make sense as a wildcard